server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -162,6 +192,7 @@ export class ScribeLambda implements IPartitionLambda {
                         const errorMsg = `Invalid message sequence number.`
                             + `Current message @${value.operation.sequenceNumber}.`
                             + `ProtocolHandler @${lastProtocolHandlerSequenceNumber}`;
+                        Lumberjack.error(errorMsg, getLumberBaseProperties(this.documentId, this.tenantId));
```

Comment:
This might not be needed (though it could be useful to have it here) as the upper layers catch the exception and log it. Same for line 148 above

Reply:
Removed the duplicate